Cargo.toml

```diff
@@ -51,6 +51,10 @@ hostname-validator = "1.1.1"
 reqwest = { version = "0", default-features = false, features = [
   "rustls-tls-native-roots",
 ] }
+rustls = { version = "0.23", default-features = false, features = [
```

Comment:
Maybe it's better to re-export on policy-fetcher and consume it that way.

Reply:
Fine by me. One less dependency to keep on sync among our code.